### ISSUE TITLE
curl test fails on CentOS 7 and Oracle Linux 8

### ISSUE THREAD

**Issue by Tux** [abeltje/Test-Smoke#74]
```
$ prove -vwb t/poster-post.t
t/poster-post.t ..
# HTTP::Daemon (6.16): IPv6 (::1)
# Temporary daemon at: http://[::1]:16573/
ok 1 - An object of class 'Test::Smoke::Poster::LWP_UserAgent' isa 'Test::Smoke::Poster::LWP_UserAgent'
ok 2 - write_json
ok 3 - Got id (LWP::Useragent: http://[::1]:16573/report)
ok 4 - An object of class 'Test::Smoke::Poster::Curl' isa 'Test::Smoke::Poster::Curl'
ok 5 - write_json
[2023-04-29 12:44:23+0200] [POST] curl exitcode: 3
not ok 6 - Got id (curl: http://[::1]:16573/report) curl v7.29.0

#   Failed test 'Got id (curl: http://[::1]:16573/report) curl v7.29.0'
#   at t/poster-post.t line 145.
#          got: '[POST]: >POST to '"http://[::1]:16573/report"' curl failed: 3
# < at /pro/3gl/CPAN/Test-Smoke-git/blib/lib/Test/Smoke/Poster/Base.pm line 148.
#       Test::Smoke::Poster::Base::post(Test::Smoke::Poster::Curl=HASH(0x3905b78)) called at t/poster-post.t line 143
#       eval {...} called at t/poster-post.t line 143
# '
#     expected: '42'
# {
#   'poster' => bless( {
#     '_curl' => bless( {
#       'arguments' => [
#         '-A',
#         'Test::Smoke/1.79_08 (Test::Smoke::Poster::Curl)',
#         '-d',
#         '@/tmp/curl-tsrepostA6Q1aE',
#         '--silent',
#         '"http://[::1]:16573/report"'
#       ],
#       'command' => '/usr/bin/curl',
#       'exitcode' => 3,
#       'verbose' => 0
#     }, 'Test::Smoke::Util::Execute' ),
#     '_curlbin' => '/usr/bin/curl',
#     '_ddir' => '/pro/3gl/CPAN/Test-Smoke-git/t',
#     '_jsnfile' => 'testsuite.jsn',
#     '_poster' => 'curl',
#     '_smokedb_url' => '"http://[::1]:16573/report"',
#     '_ua_timeout' => undef,
#     '_v' => 0
#   }, 'Test::Smoke::Poster::Curl' ),
#   'response' => '[POST]: >POST to \'"http://[::1]:16573/report"\' curl failed: 3
# < at /pro/3gl/CPAN/Test-Smoke-git/blib/lib/Test/Smoke/Poster/Base.pm line 148.
#       Test::Smoke::Poster::Base::post(Test::Smoke::Poster::Curl=HASH(0x3905b78)) called at t/poster-post.t line 143
#       eval {...} called at t/poster-post.t line 143
# '
# }
ok 7 - An object of class 'Test::Smoke::Poster::HTTP_Tiny' isa 'Test::Smoke::Poster::HTTP_Tiny'
ok 8 - write_json
ok 9 - Got id (HTTP::Tiny: http://[::1]:16573/report
ok 10 - no warnings
1..10
# tear down: 3109
# Looks like you failed 1 test of 10.
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/10 subtests

Test Summary Report
-------------------
t/poster-post.t (Wstat: 256 (exited 1) Tests: 10 Failed: 1)
  Failed test:  6
  Non-zero exit status: 1
Files=1, Tests=10,  0 wallclock secs ( 0.02 usr  0.01 sys +  0.20 cusr  0.02 csys =  0.25 CPU)
Result: FAIL
```

**Comment by abeltje** [comment]
See PR #75

**Comment by abeltje** [comment]
Fixed in 1.79_09 will be in 1.80
